agent/structs/connect_ca.go

```diff
@@ -224,6 +224,10 @@ type IssuedCert struct {
 	// AgentURI is the cert URI value.
 	AgentURI string `json:",omitempty"`
 
+	// ServerURI is the URI value of a cert issued for a server agent.
+	// The same URI is shared by all servers in a Consul datacenter.
+	ServerURI string `json:",omitempty"`
+
```

Comment:
Just curious why `IssuedCert` isn't exported from the `API` package? Looks like it's returned from at least one endpoint `/v1/agent/connect/ca/leaf/test?token=<token>`.

Reply:
@DanStough It is in the `api` package, but for some reason it has a different name there. I assume it was for clarity? https://github.com/hashicorp/consul/commit/9638466b88aab7c415fba1bf035bb37e8d4efd66

Cert type: https://github.com/hashicorp/consul/blob/8c46e48e0dafc9d474c8333a86b4d652a43a5f5a/api/connect_ca.go#L108
Endpoint client: https://github.com/hashicorp/consul/blob/8c46e48e0dafc9d474c8333a86b4d652a43a5f5a/api/agent.go#L1112

We don't need to expose `ServerURI` via that type because users can't request leaf certs for Consul servers.